snuba/web/rpc/common/aggregation.py

```diff
@@ -201,6 +242,38 @@ def get_count_column(aggregation: AttributeAggregation) -> Expression:
     return f.count(field, alias=_get_count_column_alias(aggregation))
 
 
+def _get_possible_percentiles(
+    percentile: float, granularity: float, width: float
+) -> List[float]:
+    # we multiply by 100000 to get a precision of 5 decimal places
+    return [
+        x / 100000
+        for x in range(
+            int(max(granularity, percentile - width) * 100000),
+            int(min(1, percentile + width) * 100000),
+            int(granularity * 100000),
+        )
+    ]
+
+
+def _get_possible_percentiles_expression(
+    aggregation: AttributeAggregation,
+    percentile: float,
+    granularity: float = 0.005,
+    width: float = 0.2,
```

Comment:
I did some rough calculations to figure out the width (we will never need to compute percentiles outside that range given current constraints). The granularity was arbitrarily decided. This constants aren't very important as all of these calculations are very rough and approximate. The width isn't important as long as it contains the range of values that we actually care about, I just added it as a performance optimization. The granularity might need to be adjusted, but I think this is a good starting value.

Reply:
It's important to capture these in comments, it won't always be you working on this code

Reply:
I added some comments to explain

Reply:
> The granularity was arbitrarily decided. This constants aren't very important as all of these calculations are very rough and approximate. The width isn't important as long as it contains the range of values that we actually care about, I just added it as a performance optimization. The granularity might need to be adjusted, but I think this is a good starting value.

This is a much more useful comment than what you actually wrote down in the code. Think of how someone who has no idea what any of this is will have to change the code. There's these seemingly arbitrary concepts like `width`, `granularity` and `percentile`. It's unclear how they relate to each other and what it would mean to change them